### PR TITLE
add cytoscape-panzoom v2.1.3 

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/cytoscape/cytoscape.js-panzoom
**Watch 27
  Star 11
 Fork 8**
@Piicksarn  could u help me check this PR for #6478 ? thank you!

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [ ] Notable popularity cc https://github.com/cdnjs/cdnjs/issues/6478#issuecomment-166830418
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/cytoscape-panzoom/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "filename": "cytoscape-panzoom.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cytoscape/cytoscape.js-panzoom.git"
+  },
+  "keywords": [
+    "cytoscape",
+    "cyext"
+  ],
+  "license": "LGPL-3.0+",
```
Comment (PeterDaveHello) [inline ajax/libs/cytoscape-panzoom/package.json]: Unreliable `+` should be deprecated.
https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L119

File: ajax/libs/cytoscape-panzoom/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "filename": "cytoscape-panzoom.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cytoscape/cytoscape.js-panzoom.git"
+  },
+  "keywords": [
+    "cytoscape",
+    "cyext"
+  ],
+  "license": "deprecated_LGPL-3.0+",
```
Comment (maruilian11) [inline ajax/libs/cytoscape-panzoom/package.json]: @PeterDaveHello is that ok?

Comment (PeterDaveHello) [inline ajax/libs/cytoscape-panzoom/package.json]: yep

**Comment by maruilian11** [comment]
@Piicksarn could you help check this PR? thank you~

**Pull request merged**